internal/action/command.go

```diff
@@ -782,7 +782,8 @@ func (h *BufPane) GotoCmd(args []string) {
 				InfoBar.Error(err)
 				return
 			}
-			h.GotoLoc(buffer.Loc{0, line})
+			col := util.Clamp(h.Buf.GetActiveCursor().X, 0, util.CharacterCount(h.Buf.LineBytes(line)))
+			h.GotoLoc(buffer.Loc{col, line})
```

Comment:
I don't think it's a good idea. When I jump to another line without specifying the column, I expect to be at the beginning of the line, not at some unrelated position somewhere in the middle of it.

Reply:
I've no problem removing this, but let me at least introduce my intention:

The scenario was a large LUT/configuration table which will be modified at a dedicated column. Now someone would like to modify exactly the same column in a different line, but is too lazy to type the column again.

But yes, if this isn't common sense then we can simply drop this commit.

Reply:
I think jumping to the beginning of a line is a more common use case (for example, think of jumping to a line specified in a compiler error message). Your use case seems to be quite specific. For instance, am I right that your use case assumes that there are no tab characters in the file, or if there are, they are all at the same positions in each line (so that when moving to another line, if the logical column number doesn't change, the visual column location doesn't change either)?

By the way... I've just realized that relative column numbers might not make a lot of sense either, for similar reasons. Are there real uses for relative column numbers? If not, maybe for simplicity it's better to stick to relative line numbers only? I.e. support `+`, `-` and `~` for lines only, and for columns support only specifying either absolute column, or no column at all (i.e. don't extend the existing functionality for column numbers). @dase78 what do you think?

Reply:
I also would only expact relative movement only in lines not in columns. I guess most people are more used to the word by word movement in the line. At least I never move in another way in the lines.

But I am no coder.

Reply:
But we don't force anybody to use it. It exists, it's somehow generic even from documentation point of view.
Otherwise we've to described in the documentation that relative jumps are only supported for lines, but not for columns.
(Huh, why? The input is of integer type too and prefix handling already exists...)

At least for me it's one example for consistency instead of exceptions.

But I'm the last one who will argue against the community alone.
If you don't think that it's a kind of common theme, then I will remove it too...with a sad eye. :wink:

Reply:
> But we don't force anybody to use it. It exists, it's somehow generic even from documentation point of view. 

+1

Reply:
Well, I don't insist, I'm just not the one who thinks that adding features "just in case" is a wise thing to do. We don't know if the feature is gonna be used by anyone at all, so we don't know how is it gonna be used, so we don't know what is the best way to implement it (just because we don't really know what for)... And what if someone in the future would like to add some other (actually useful) feature to the `goto` command, but finds that its syntax is already overloaded...

Nevertheless, I'd be happy to hear any arguments about potential usefulness of relative columns.

> Otherwise we've to described in the documentation that relative jumps are only supported for lines, but not for columns.

In our case it's quite the opposite: now we need to describe that `~` is only supported for lines, unlike `+/-`. (You haven't done that in your documentation, but you should have.) Whereas without relative columns, we could just say `Line can be prefixed with +, - or ~...` without mentioning columns. :)